cts/cli/regression.crm_resource.exp

```diff
@@ -0,0 +1,569 @@
+=#=#=#= Begin test: Check locations and constraints for prim1 =#=#=#=
+=#=#=#= End test: Check locations and constraints for prim1 - OK (0) =#=#=#=
+* Passed: crm_resource   - Check locations and constraints for prim1
+=#=#=#= Begin test: Recursively check locations and constraints for prim1 =#=#=#=
+=#=#=#= End test: Recursively check locations and constraints for prim1 - OK (0) =#=#=#=
+* Passed: crm_resource   - Recursively check locations and constraints for prim1
+=#=#=#= Begin test: Check locations and constraints for prim1 in XML =#=#=#=
+<pacemaker-result api-version="X" request="/home/clumens/src/pacemaker/tools/.libs/crm_resource -a -r prim1 --output-as=xml">
```

Comment:
I don't think "request" will be the same on CI :)

Reply:
Heh, whoops.  Our regex in cts/cts-cli.in needs to be updated to include crm_resource.  Fixed.